controllers/recipes.js

```diff
@@ -199,21 +199,34 @@ module.exports = {
     }
   },
   deleteRecipe: async (req, res) => {
-    console.log('deleteRecipe was invoked')
+    console.log('   deleteRecipe was invoked')
     try {
       const recipeDbId = req.params.id;
       console.log("   Finding recipe id: ", recipeDbId);
+      const favoritedRecipes = await Favorite.find({ recipe: recipeDbId });
+      console.log("   Checking if favorited:", favoritedRecipes);
+      if (favoritedRecipes.length === 0) {
+        // The recipe does not exist in the favorites collection
+        console.log("   Recipe is not favorited");
+      } else {
+        // The recipe exists in the favorites collection
+        console.log("   Recipe is favorited", `   Here are the ids: ${favoritedRecipes}`);
```

Comment:
This is printing out the entire Favorite object and not just the ids like it says. Not a super important thing since this is mostly for debugging/visibility during development but maybe change the verbiage to "Here are the Favorite documents" or just "Favorite documents"?

Reply:
Good catch.